automation/jenkins_build.sh

```diff
@@ -334,7 +342,7 @@ deploy_to_s3() {
 			su deployer<<EOSU
 echo "${BUILD_VERSION}" > "/host/images/${SLUG}/latest"
 /usr/src/app/node_modules/.bin/coffee /usr/src/app/scripts/prepare.coffee
-if [ -z "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/)" ] || [ -n "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/IGNORE)" ]; then
+if [ -z "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/)" ] || [ -n "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/IGNORE)" ] || [ "$deployTo" = "multistaging" ]; then
```

Comment:
Is the check where to deploy really necessary? We should not depend on that. The code check the appropriate S3 bucket to see if the image has been deployed there and acts accordingly